materials/internals/reset.mod.css

```diff
@@ -9,24 +9,9 @@
 html,
 body {
   color: var(--color-black);
-  font-family: 'Open Sans', sans-serif;
   font-size: 13px;
```

Comment:
Unrelated but should we consider using `rem` for sizes instead of hard-coded pixels?